NEWS.md

```diff
@@ -1,3 +1,17 @@
+# EpiNow2 1.3.3.9000
```

Comment:
This was 1.3.4 in the original PR.

Reply:
It was meant as indicating dev version as recommended at https://r-pkgs.org/release.html#post-release to be eventually changed into whichever release version this ends up being.